coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -182,47 +156,63 @@ <h2 class="section-label">Testimonials</h2>
         <div>
           <div class="card-tabs">
             <div class="row">
-              <div class="testimonials-card-tabs">
-                <ul class="tabs tabs-fixed-width">
-                  <li class="tab"><a class="active" href="#testimonial1">adrian</a></li>
-                  <li class="tab"><a href="#testimonial2">rachal</a></li>
-                  <li class="tab"><a href="#testimonial3">mike</a></li>
-                </ul>
-              </div>
+            <div class="testimonials-card-tabs">
+            <ul class="tabs tabs-fixed-width">
+              <li class="tab"><a class="active" href="#testimonial1">STUDENT: NAOMI</a></li>
+              <li class="tab"><a href="#testimonial2">TRAINER: NKECHI /FADWA</a></li>
+              <li class="tab"><a href="#testimonial3">JOURNO: NANCY AGUTU (FROM THE STAR)</a></li>
+              <li class="tab"><a href="#testimonial4">UNIVERSITY: TR.WAMBUI</a></li>
+              <li class="tab"><a href="#testimonial5">NEWSROOM: NAN/ AWOKO/STAR NEWSPAPER</a></li>
+            </ul>
+          </div>
             </div>
           </div>
           <div class="card-content grey lighten-4">
             <div id="testimonial1" class="testimonial-div">
               <div class="testimonial-card">
                 <h3>Words to live by</h3>
-                <p>Adrian Simpson set up a basic website for his new speechwriting business and
-                  began
-                  advertising online through search.
+                <p>“StoryLab Academy has provided an avenue to learn courses not offered in my school. Given that I am a writer, courses like verification will enable me write better stories.”
```

Comment:
@Sharonsyra how about &ldquo and &rdquo ?

Reply:
Good idea let me use that.